backend/aws/lambda/create_response.py

```diff
@@ -0,0 +1,34 @@
+import json
+
+def trim_response(error, results):
+    if len(results) == 0:
+        raise('No results returned, should include at least source')
+    
+    shortened_results = [{'link': result['link'], 'source': result['source'], 'score': result['score']} for result in results]
+    source = shortened_results[0]
+    shortened_results = shortened_results[1:]
+    sorted_results = sorted(shortened_results, key=lambda k: k['score'], reverse=True)
+
+    return json.dumps([source]+sorted_results[:3])
```

Comment:
Why `:3`? Is this top 3?

Reply:
Yah, that is what that syntax means, but it should be a config value, not just hard coded top 3